### ISSUE TITLE
1.20.1 Newest version forces "safe mode"

### ISSUE THREAD

**Issue by FluffiKiki** [MehVahdJukaar/amendments#84]
Minecraft 1.20.1, Forge 47.2.20, Amendments 1.1.31

Newest version causes loaded world to enter "Safe mode" or go back to main menu. 
[latest.log](https://github.com/MehVahdJukaar/amendments/files/15415699/latest.log)

**Comment by MehVahdJukaar** [comment]
That file is huge. Does it say anything about datapack load failed? Should say why

**Comment by MehVahdJukaar** [comment]
Also are you using latest moonlighr version

**Comment by FluffiKiki** [comment]
Yeah, I had someone else read it for me and it said that Amendments was the reason it failed. Wasn't using the latest version, will test it when I have the time

**Comment by MehVahdJukaar** [comment]
your issue is 100% outdated moonlight
